source/2015-03-25-ten-tools-to-help-build-automated-marketing-websites.html.markdown

```diff
@@ -0,0 +1,123 @@
+---
+title: Ten tools to help build automated marketing websites.
+subtitle: Marketing automation tools for the rest of us.
+date: 2015-03-25 06:06 +0000
+tags: seo, sem, automated marketing
+category: marketing
+author:
```

Comment:
@johnkoht The idea was to allow guest posts, I think abstracting the frontmatter makes this more cumbersome and also will bloat the repo if we get a lot of guest bloggers.